.github/workflows/pkgci_regression_test.yml

```diff
@@ -356,10 +356,10 @@ jobs:
             --goldentime-rocm-e2e-ms 325.0 \
             --goldentime-rocm-unet-ms 77.0 \
             --goldentime-rocm-clip-ms 15.5 \
-            --goldentime-rocm-vae-ms 74.0 \
-            --goldendispatch-rocm-unet 1714 \
+            --goldentime-rocm-vae-ms 85.2 \
+            --goldendispatch-rocm-unet 1713 \
```

Comment:
Would ideally like to know why this particular program went from 74ms to 85ms, if we trust the benchmarking methodology here enough.

Reply:
Fair enough. This has been on my mind quite a bit lately. I'd like to strike a good balance between core/upstream sanity (default flags must work, medium sized test suites including XFAILs) and early signal for bleeding edge usage that contributors are iterating on.

I'm trying to rework our CI jobs such that the "core" uses standard runners and baseline configurations then each group contributing to the project (e.g. AMD, ARM, Google) can donate CI runners and add limited sets of optional tests/benchmarks that run on PRs/commits. For heavier tests or workflows needing nonstandard configuration (Android phones, wasm/webgpu-capable web browsers, laptops, etc.) those _might_ be able to find a place in-tree but they should probably live out of tree or in private repos (especially for unreleased hardware ;P).

Reply:
I'm going to try to figure this out, but I probably won't again - this benchmark stuff has a bad UX and takes way too much effort to deal with to be on presubmit.

Reply:
RE this being a benchmark - I think that's ok and a good heads up like you say - but there's different tiers for workflows: something using so many non-standard flags carefully crafted to produce a specific result by a downstream project and then requiring that we match within microseconds from any change in the compiler pipeline is not possible. If it's a change detector that's one thing (though still noisy and annoying to daily development), but using all those non-default flags is YMMV and downstream things need to remember that and upstream fixes, etc - we can't have a compiler with an infinite number of potential combinations that never regresses anything.